applications/luci-app-siitwizard/luasrc/model/cbi/siitwizard.lua

```diff
@@ -49,8 +49,8 @@ ula = ula:add(find_ll())
 
 
 -------------------- View --------------------
-f = SimpleForm("siitwizward", "SIIT-Wizzard",
- "This wizzard helps to setup SIIT (IPv4-over-IPv6) translation according to RFC2765.")
+f = SimpleForm("siitwizard", "SIIT-Wizard",
```

Comment:
Great catch on typo in the form name! The readability of `set up SIIT` makes me think it should also be `set up <noun>` in other cases in this PR.

Reply:
Readability... well, that's ofc important. :) 

"Set up x" and "set x up" are both common forms of phrasal verbs separated by the object, acceptable in English. 
https://dictionary.cambridge.org/dictionary/english/set-up